windows/OrientationWindows/OrientationWindows.vcxproj

```diff
@@ -155,11 +156,6 @@
   <ItemGroup>
     <Midl Include="ReactPackageProvider.idl" />
   </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="$(ReactNativeWindowsDir)\Microsoft.ReactNative\Microsoft.ReactNative.vcxproj">
```

Comment:
I believe this was only originally needed before 
Microsoft.ReactNative.Uwp.CppLib.props and Microsoft.ReactNative.Uwp.CppLib.targets was added. And those were added long ago.